environment_forge_full.yml

```diff
@@ -0,0 +1,40 @@
+name: qcodesforge
+channels:
+ - conda-forge
+dependencies:
+ - python=3.7.9
+ - pip
+ - deepdiff
+ - numpy=1.19.5
```

Comment:
Since there is no way for dependabot to automatically update the conda deps I think we should leave all of them unpinned except python. I would probably also move this file to within the .github dir to make it clear that it is not intended for end users.